service/controller/resource/prometheus/resource.go

```diff
@@ -153,7 +153,7 @@ func toPrometheus(v interface{}, config Config) (metav1.Object, error) {
 	}
 
 	image := fmt.Sprintf("%s/giantswarm/prometheus:%s", config.Registry, config.PrometheusVersion)
-
+	pageTitle := fmt.Sprintf("%s Prometheus", key.ClusterID(cluster))
```

Comment:
I'd vote for including the installation name and no `'s` (like in tests), so e.g. `gauss/s06tw Prometheus`.